### DISCUSSION TITLE
Feedback session at 20220708

### DISCUSSION THREAD

**Discussion by taijusanagi** [kaguraswap/monorepo#142] [General]
Any feedback is welcome!!

**Comment by ksuhara** [comment]
better to have some guide on top page

**Reply by taijusanagi** [reply]
like this 

<img width="1405" alt="Screen Shot 2022-07-08 at 16 39 40" src="https://user-images.githubusercontent.com/38043569/177942358-23faf980-90b7-4c56-8d6f-2b8d7140d3d8.png">

**Reply by taijusanagi** [reply]
https://github.com/kaguraswap/monorepo/issues/143

**Comment by taijusanagi** [comment]
discord seems personal

**Reply by taijusanagi** [reply]
https://github.com/kaguraswap/monorepo/issues/144

**Comment by taijusanagi** [comment]
<img width="449" alt="Screen Shot 2022-07-08 at 16 17 25" src="https://user-images.githubusercontent.com/38043569/177938359-6e5099ac-4044-407d-80b4-1b691014bb2e.png">

my page big

no padding with copy

**Reply by taijusanagi** [reply]
https://github.com/kaguraswap/monorepo/issues/146

**Comment by taijusanagi** [comment]
improve design

<img width="1152" alt="Screen Shot 2022-07-08 at 16 20 45" src="https://user-images.githubusercontent.com/38043569/177938941-51120f90-3b76-4c1f-9920-f7983e4fdf2d.png">

**Reply by taijusanagi** [reply]
chain is difficult understand

this is good for telling which chan the asset is in

<img width="560" alt="Screen Shot 2022-07-08 at 16 24 11" src="https://user-images.githubusercontent.com/38043569/177939552-b944bad2-85a5-433f-a171-87db2361f22d.png">

**Reply by taijusanagi** [reply]
chain id here too

<img width="409" alt="Screen Shot 2022-07-08 at 16 25 35" src="https://user-images.githubusercontent.com/38043569/177939798-0653f03d-1738-4156-b3a2-e1b05bffc142.png">

**Reply by taijusanagi** [reply]
put this in dropdown

<img width="323" alt="Screen Shot 2022-07-08 at 16 25 58" src="https://user-images.githubusercontent.com/38043569/177939952-6a5ee4e3-f25a-4dff-9a9b-3eb7842d293b.png">

**Comment by taijusanagi** [comment]
check with non 350 * 350 image

**Reply by taijusanagi** [reply]
this is about ratio

**Comment by taijusanagi** [comment]
I want default amount

<img width="452" alt="Screen Shot 2022-07-08 at 16 21 59" src="https://user-images.githubusercontent.com/38043569/177939163-d21dee09-6a52-45f7-9db5-7637d7f8618e.png">

**Comment by taijusanagi** [comment]
Fee can be short

<img width="454" alt="Screen Shot 2022-07-08 at 16 22 20" src="https://user-images.githubusercontent.com/38043569/177939228-95cb685b-068b-46d9-b991-9fe7fe13a0c2.png">

**Comment by taijusanagi** [comment]
trigger switch network here

<img width="454" alt="Screen Shot 2022-07-08 at 16 23 44" src="https://user-images.githubusercontent.com/38043569/177939469-f1c0399e-f845-491f-8ed7-67fd6a9ac376.png">

**Comment by taijusanagi** [comment]
Fee is not working, this is related with BSP

https://github.com/kaguraswap/monorepo/commit/04aa1951335f95827fe25852b9afd370a360e23f

**Reply by taijusanagi** [reply]
offer is working but sell is not working

**Comment by taijusanagi** [comment]
Add link to the proper order

<img width="561" alt="Screen Shot 2022-07-08 at 16 31 31" src="https://user-images.githubusercontent.com/38043569/177940829-6a7cbaad-6832-4e0e-a335-2cc153539124.png">

**Comment by taijusanagi** [comment]
Implement currency

<img width="449" alt="Screen Shot 2022-07-08 at 16 31 59" src="https://user-images.githubusercontent.com/38043569/177940892-a3294daf-a353-406a-9e4e-210b6c9d3327.png">

**Comment by taijusanagi** [comment]
status handling

<img width="547" alt="Screen Shot 2022-07-08 at 16 34 39" src="https://user-images.githubusercontent.com/38043569/177941400-5611e662-bf6a-457e-8842-9cca7d616c06.png">

**Comment by taijusanagi** [comment]
balance may be not required and it is not working some time.

<img width="312" alt="Screen Shot 2022-07-08 at 16 35 19" src="https://user-images.githubusercontent.com/38043569/177941524-08ce0da9-2023-425b-8fb4-b7b3ad3f3646.png">

**Comment by Kentaroyamamura** [comment]
footer等にPowered by Seaport的なのがあっても良いかも

**Reply by taijusanagi** [reply]
how about adding protocols in footer?

**Comment by Aqrare** [comment]
When enter the  URL of Opensea, the chainId doesn't be appeared

<img width="451" alt="Screen Shot 2022-07-08 at 16 43 28" src="https://user-images.githubusercontent.com/64068653/177942975-79dfc87d-b43a-4ed9-9b9b-019c9ad53560.png">

**Reply by taijusanagi** [reply]
https://github.com/kaguraswap/monorepo/issues/116

**Comment by Kentaroyamamura** [comment]
- 手数料0
- OTC
- middlemanいないよ
とかの特徴くらいはキャッチコピー的に欲しいかも

**Reply by taijusanagi** [reply]
How about setting concept in top page

**Comment by Aqrare** [comment]
<img width="452" alt="Screen Shot 2022-07-08 at 16 45 54" src="https://user-images.githubusercontent.com/64068653/177943414-cd831b3f-fc6c-4d9b-9ac4-3b8aef05891f.png">

細かいけどTotal Costの桁数を揃えたい

**Reply by taijusanagi** [reply]
https://github.com/kaguraswap/monorepo/discussions/142#discussioncomment-3105552

**Comment by Kentaroyamamura** [comment]
1) Connect Wallet
2) Create Order 
3) Swap
とかの3stepとかでKagura Swapでできることを示すとかしてあげると親切な気がするのと、SudoSwapとの違い等が暗示できるような特徴を書けると良いかも。

**Reply by taijusanagi** [reply]
How about setting operation guide in top?

**Comment by taijusanagi** [comment]
How about bringing the concept of tip??

**Reply by taijusanagi** [reply]
I want 
- tip is set as 2.5 by initial.
- use can increase and decrease
- use also change the address

**Comment by Kentaroyamamura** [comment]
手数料が0が無い事が最大の特徴で、それが利用者に還元されているという考え方ができるのであればその総額をドTOPに"TotalでいくらNFTファンに還元したよ！"というのがあっても良いかも。
